test/error.jl

```diff
@@ -160,3 +160,5 @@ using .Issue54029
     str = sprint(showerror, ce)
     @test str == "something sad has happened even earlier\n\n...and 2 more exceptions.\n"
 end
+
+@test_throws ErrorException error()
```

Comment:
This belongs to a different PR, right? It's best if the changes from a PR are self-contained, please drop the commit from here